source/OutfitterPanel.cpp

```diff
@@ -371,15 +371,15 @@ ShopPanel::BuyResult OutfitterPanel::CanBuy(bool onlyOwned) const
 		if(isInStore)
 		{
 			// Player hit 'i' or 'c' when they should've hit 'b'.
-			if(playerShip)
+			if(shipSelection.Selected())
```

Comment:
you are calling the getter a few times inside this function, how about you store it in a variable?